nuget/lib/dependabot/nuget/nuget_helpers.rb

```diff
@@ -0,0 +1,108 @@
+# typed: true
+# frozen_string_literal: true
+
+require "dependabot/nuget/cache_manager"
+require "dependabot/nuget/update_checker/repository_finder"
+
+module Dependabot
+  module Nuget
+    class NugetHelpers
```

Comment:
Like how you gathered a lot of the v3 api usage together here. Would love to see us do the same for v2.

Reply:
I almost pulled V2 in in this PR, but it's already quite large and we're wanting to get it out to people soon. If anyone were to move the v2 APIs onto here in the future (and even better combine them so the repository_details decides which one to use) I'd be super here for it.